packages/models/src/utils/decorator/provider.ts

```diff
@@ -144,3 +144,14 @@ export const DefaultLock =
 export const Omnilock = (): ClassDecorator => DefaultLock('OMNILOCK')
 
 export const Secp256k1Lock = (): ClassDecorator => DefaultLock('SECP256K1_BLAKE160')
+
+/*
+ * Alias pattern to filter for better understanding in functional context
+ */
+export {
+  Pattern as Filter,
```

Comment:
Why not change the `Pattern` name when defined?

Reply:
> Why not change the `Pattern` name when defined?

It will be a breaking change that some users who have tried it will be bothered.

Reply:
`Pattern` can be considered a term used in an internal context, in fact, it was referenced in several documents and discussions. `Filter` can be considered as an user-oriented keyword for developers who are familiar to functional programming